src/web/app/src/components/ThemeSelector.tsx

```diff
@@ -0,0 +1,133 @@
+// eslint-disable-next-line no-use-before-define
+import React, { useState } from 'react';
```

Comment:
```suggestion
import { useState } from 'react';
```

Reply:
This wasn't compiled for me. Somehow it works now.

Reply:
Sorry I think it wasn't a compile issue. Actually it was a lint error. It still gives me `'React' was used before it was defined  no-use-before-define` error.

Reply:
My bad, it turned out that my linter wasn't working, but thanks to this I found a bug in our `vscode` settings. You can ignore this comment.